tracks/python/exercises/perfect-numbers/mentoring.md

```diff
@@ -29,7 +29,7 @@ A simple optimization is to use `range(1, number // 2 + 1)` as the `for` loop `i
 This halves the complexity of the solution.
 
 For a more efficient solution, one can compute all the factors by using the smaller `range(1, int(math.sqrt(number)) + 1)` as the `iterable`.
-This solution is longer and more involved but significantly faster, reducing the complextiy from `O(n)` (_linear_) to `O(sqrt(n))` (_square root_).
+This solution is longer and more involved but significantly faster, reducing the complexities from `O(n)` (_linear_) to `O(sqrt(n))` (_square root_).
```

Comment:
"complexity" here should be singular, not plural.

Reply:
ok I will make the changes